infra/stack/eks.tf

```diff
@@ -0,0 +1,69 @@
+module "label" {
+  source = "cloudposse/label/null"
+
+  name       = var.label_name
+  attributes = ["cluster"]
+}
+
+locals {
+  # Prior to Kubernetes 1.19, the usage of the specific kubernetes.io/cluster/* resource tags below are required
+  # for EKS and Kubernetes to discover and manage networking resources
+  # https://www.terraform.io/docs/providers/aws/guides/eks-getting-started.html#base-vpc-networking
+  tags = { "kubernetes.io/cluster/${module.label.id}" = "shared" }
+}
+
+module "vpc" {
+  source = "cloudposse/vpc/aws"
+
+  cidr_block = "172.16.0.0/16"
+
+  tags    = local.tags
+  context = module.label.context
+}
+
+module "subnets" {
+  source = "cloudposse/dynamic-subnets/aws"
+
+  availability_zones   = var.availability_zones
+  vpc_id               = module.vpc.vpc_id
+  igw_id               = module.vpc.igw_id
+  cidr_block           = module.vpc.vpc_cidr_block
+  nat_gateway_enabled  = true
+  nat_instance_enabled = false
+
+  tags    = local.tags
+  context = module.label.context
+}
+
+module "eks_cluster" {
+  source = "cloudposse/eks-cluster/aws"
```

Comment:
`cloudposse` are great and their modules are top-notch, but usually I prefer to use official ones when available , i.e. https://github.com/terraform-aws-modules/terraform-aws-eks - no need to change this though, was just an FYI!

Reply:
Sure, will try using the official one's going forward.